configs/generate.js

```diff
@@ -28,18 +26,17 @@ const generateStoryblokEventRoutes = async () => {
 
     return ret
   } catch (e) {
+    // eslint-disable-next-line no-console
```

Comment:
i prefer disable console.warn and error lint but let error in console.log

Reply:
Done!